### PR TITLE
Issue #2557 Integration: Make container status function non-fragile

### PR DESCRIPTION
**Pull request by praveenkumar**

### CODE REVIEW

File: test/integration/util/util.go [hunk 1]
```diff
@@ -247,8 +247,8 @@ func (m *MinishiftRunner) GetProfileList() string {
 	return strings.Trim(cmdOut, " \n")
 }
 
-func (m *MinishiftRunner) GetOpenshiftContainers() string {
-	cmdOut, _, _ := m.RunCommand("ssh -- docker ps")
+func (m *MinishiftRunner) GetOpenshiftContainers(containerName string) string {
+	cmdOut, _, _ := m.RunCommand(fmt.Sprintf(`ssh -- docker ps -f "name=%s"`, containerName))
```
Comment (LalatenduMohanty) [inline test/integration/util/util.go]: 👍

File: test/integration/features/addon-registry-route.feature [hunk 2]
```diff
@@ -24,7 +24,7 @@ so that user can able to push their container images directly to registry and de
       And evaluating stdout of the previous command in host shell
       And executing "minishift oc-env" in host shell succeeds
       And evaluating stdout of the previous command in host shell
-     Then with up to "20" retries with wait period of "3" seconds container image "origin-docker-registry" should be "running"
+     Then with up to "20" retries with wait period of "3" seconds container name "registry_docker-registry" should be "running"
```
Comment (LalatenduMohanty) [inline test/integration/features/addon-registry-route.feature]: The name `registry_docker-registry` looks odd.

Comment (praveenkumar) [reply test/integration/features/addon-registry-route.feature]: @LalatenduMohanty It is not odd, if you check the container name which openshift creates is something like `k8s_registry_docker-registry-1-vxt5j_default_a720081a-8012-11e8-984c-5254001ed95b_1` for the registry.

Comment (LalatenduMohanty) [reply test/integration/features/addon-registry-route.feature]: @praveenkumar Why not using the complete name `k8s_registry_docker-registry` ?

Comment (praveenkumar) [reply test/integration/features/addon-registry-route.feature]: @LalatenduMohanty Has is going to change always so you need to use filter and only use what you for sure not going to change, same trick we are using for origin container status for patch set.

Comment (LalatenduMohanty) [reply test/integration/features/addon-registry-route.feature]: @praveenkumar right thats why I was suggesting `k8s_registry_docker-registry`
```
$ docker ps -f name=k8s_registry_docker-registry
CONTAINER ID        IMAGE               COMMAND                  CREATED              STATUS              PORTS               NAMES
6bf509f27473        cf67b3158bfb        "/bin/sh -c '/usr/..."   About a minute ago   Up About a minute                       k8s_registry_docker-registry-1-vh5pq_default_dde2153b-8026-11e8-b089-f23a1db36513_0
```

**Pull request merged**